src/PhpGenerator/Method.php

```diff
@@ -392,4 +404,23 @@ public function setNamespace(PhpNamespace $val = NULL)
 		return $this;
 	}
 
+	/**
+	 * @param  string|NULL
+	 * @return self
+	 */
+	public function setReturnType($val)
```

Comment:
Please change `$returnType` from `string|NULL` to `string` to be compatible with rest of API. (I know it is not ideal.)